core/src/main/java/org/infinispan/configuration/cache/IndexingConfigurationBuilder.java

```diff
@@ -32,8 +33,23 @@
       attributes = IndexingConfiguration.attributeDefinitionSet();
    }
 
-   public boolean enabled() {
-      return attributes.attribute(INDEX).get().isEnabled();
+   public IndexingConfigurationBuilder setEnabled(boolean enabled) {
```

Comment:
As I commented before, `is` goes against 'tradition'. So does `set`. It seems the fathers of infinispan left us the tradition of doing just ```IndexingConfigurationBuilder enabled(boolean enabled)```